app/views/feeds/_form_auth_token.html.erb

```diff
@@ -1,15 +1,21 @@
-<div class="row">
-  <div class="col mb-4">
-    <div class="form-floating">
-      <%= form.text_field :label %>
-      <%= form.label :label %>
+<div class="mb-4 prx-field-group">
+  <div class="row">
+    <div class="col-6">
+      <div class="form-floating">
+        <%= form.text_field :label %>
+        <%= form.label :label %>
+      </div>
     </div>
-  </div>
 
-  <div class="col mb-4">
-    <div class="form-floating">
-      <%= form.text_field :token %>
-      <%= form.label :token %>
+    <div class="col-6">
+      <div class="form-floating">
+        <%= form.text_field :token %>
```

Comment:
Related to an earlier suggestion: if `form.object.new_record?` ... don't render the token field.  Since we auto-generate it anyways, you don't need to provide it.